try.go

```diff
@@ -70,62 +69,150 @@
 //		return nil
 //	}
 //
+//
+// Quick tour of the API
+//
+//
+// The E family of functions all remove a final error return, panicking if non-nil.
+//
+//
+// Handle allows easy assignment of that error to a return error value.
+//
+//  func f() (err error) {
+//  	defer try.Handle(&err)
+//
```

Comment:
Nit: The lack of a closing `}` makes my brain twitch 😆. Add it?

```go
func f() (err error) {
	defer try.Handle(&err)
	...
}
```

Reply:
Do it here and below.

Reply:
Hah. Done.